CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.53.2] - 2021-04-16
+### Removed
+- Disable `rpc-caller-procedure` header temporarily by stop propoagating CallerProcedure in the outbound middleware.
```

Comment:
Only saw it once, but this should be

```
Disable `rpc-caller-procedure` header temporarily by stopping propagating `CallerProcedure` in the outbound middleware.
```
-> typo in propoagating
-> by stop to by stopping

Reply:
Rephrased it: `Disable rpc-caller-procedure header temporarily by stopping the CallerProcedure propagation.`